### ISSUE TITLE
EmptyError: no elements in sequence  after fresh build of project

### ISSUE THREAD

**Issue by stephengardner** [DavideViolante/Angular-Full-Stack#137]
A fresh build of the project started up just fine.  After clicking on a link in the header such as `cats` or `login`, I receive this error.

It is from a fresh build.

Here's my console:

```
core.js:3565 Angular is running in the development mode. Call enableProdMode() to enable the production mode.
zone.js:2933 XHR finished loading: GET "http://localhost:4200/sockjs-node/info?t=1512273642441".
scheduleTask @ zone.js:2933
ZoneDelegate.scheduleTask @ zone.js:411
Zone.scheduleTask @ zone.js:236
Zone.scheduleMacroTask @ zone.js:259
(anonymous) @ zone.js:2966
proto.(anonymous function) @ zone.js:1366
AbstractXHRObject._start @ sockjs.js:1601
(anonymous) @ sockjs.js:1490
ZoneDelegate.invokeTask @ zone.js:425
Zone.runTask @ zone.js:192
ZoneTask.invokeTask @ zone.js:499
ZoneTask.invoke @ zone.js:488
timer @ zone.js:2040
setTimeout (async)
scheduleTask @ zone.js:2056
ZoneDelegate.scheduleTask @ zone.js:411
Zone.scheduleTask @ zone.js:236
Zone.scheduleMacroTask @ zone.js:259
(anonymous) @ zone.js:2072
proto.(anonymous function) @ zone.js:1366
AbstractXHRObject @ sockjs.js:1489
XHRLocalObject @ sockjs.js:2910
InfoAjax @ sockjs.js:356
InfoReceiver._getReceiver @ sockjs.js:536
InfoReceiver.doXhr @ sockjs.js:556
(anonymous) @ sockjs.js:525
ZoneDelegate.invokeTask @ zone.js:425
Zone.runTask @ zone.js:192
ZoneTask.invokeTask @ zone.js:499
ZoneTask.invoke @ zone.js:488
timer @ zone.js:2040
setTimeout (async)
scheduleTask @ zone.js:2056
ZoneDelegate.scheduleTask @ zone.js:411
Zone.scheduleTask @ zone.js:236
Zone.scheduleMacroTask @ zone.js:259
(anonymous) @ zone.js:2072
proto.(anonymous function) @ zone.js:1366
InfoReceiver @ sockjs.js:524
SockJS @ sockjs.js:730
socket @ socket.js:9
(anonymous) @ client:200
(anonymous) @ client:242
../../../../webpack-dev-server/client/index.js?http://0.0.0.0:0 @ vendor.bundle.js:798
__webpack_require__ @ inline.bundle.js:55
0 @ main.bundle.js:288
__webpack_require__ @ inline.bundle.js:55
webpackJsonpCallback @ inline.bundle.js:26
(anonymous) @ main.bundle.js:1
core.js:1350 ERROR Error: Uncaught (in promise): EmptyError: no elements in sequence
EmptyError: no elements in sequence
    at new EmptyError (EmptyError.js:28)
    at FirstSubscriber._complete (first.js:154)
    at FirstSubscriber.Subscriber.complete (Subscriber.js:122)
    at MergeMapSubscriber._complete (mergeMap.js:150)
    at MergeMapSubscriber.Subscriber.complete (Subscriber.js:122)
    at MapSubscriber.Subscriber._complete (Subscriber.js:140)
    at MapSubscriber.Subscriber.complete (Subscriber.js:122)
    at EmptyObservable._subscribe (EmptyObservable.js:83)
    at EmptyObservable.Observable._trySubscribe (Observable.js:172)
    at EmptyObservable.Observable.subscribe (Observable.js:160)
    at new EmptyError (EmptyError.js:28)
    at FirstSubscriber._complete (first.js:154)
    at FirstSubscriber.Subscriber.complete (Subscriber.js:122)
    at MergeMapSubscriber._complete (mergeMap.js:150)
    at MergeMapSubscriber.Subscriber.complete (Subscriber.js:122)
    at MapSubscriber.Subscriber._complete (Subscriber.js:140)
    at MapSubscriber.Subscriber.complete (Subscriber.js:122)
    at EmptyObservable._subscribe (EmptyObservable.js:83)
    at EmptyObservable.Observable._trySubscribe (Observable.js:172)
    at EmptyObservable.Observable.subscribe (Observable.js:160)
    at resolvePromise (zone.js:824)
    at resolvePromise (zone.js:795)
    at eval (zone.js:873)
    at ZoneDelegate.invokeTask (zone.js:425)
    at Object.onInvokeTask (core.js:4621)
    at ZoneDelegate.invokeTask (zone.js:424)
    at Zone.runTask (zone.js:192)
    at drainMicroTaskQueue (zone.js:602)
    at ZoneTask.invokeTask [as invoke] (zone.js:503)
    at invokeTask (zone.js:1540)
defaultErrorLogger @ core.js:1350
ErrorHandler.handleError @ core.js:1411
next @ core.js:5377
schedulerFn @ core.js:4223
SafeSubscriber.__tryOrUnsub @ Subscriber.js:247
SafeSubscriber.next @ Subscriber.js:194
Subscriber._next @ Subscriber.js:133
Subscriber.next @ Subscriber.js:97
Subject.next @ Subject.js:64
EventEmitter.emit @ core.js:4203
(anonymous) @ core.js:4652
ZoneDelegate.invoke @ zone.js:392
Zone.run @ zone.js:142
NgZone.runOutsideAngular @ core.js:4578
onHandleError @ core.js:4652
ZoneDelegate.handleError @ zone.js:396
Zone.runGuarded @ zone.js:158
_loop_1 @ zone.js:702
api.microtaskDrainDone @ zone.js:711
drainMicroTaskQueue @ zone.js:610
ZoneTask.invokeTask @ zone.js:503
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
```

**Comment by Thavarajan** [comment]
Not sure possibly  this is also a reason
try with full routing
https://github.com/angular/angular-cli/issues/8724

**Comment by Thavarajan** [comment]
The fix will be downgrade the  rx js to 5.5.2 
it will solve the issue

**Comment by DavideViolante** [comment]
https://stackoverflow.com/questions/41131476/emptyerror-no-elements-in-sequence
